src/features/RecipeEdit/RecipeAddController.tsx

```diff
@@ -36,12 +39,16 @@ export const RecipeAddController = () => {
     };
 
     const handleSave = (recipe: DraftRecipe) => {
-        createRecipe(recipe).then((result) => {
-            const id = result.data?.library?.createRecipe.id;
-            id
-                ? history.push(`/library/recipe/${id}`)
-                : history.push(`/library`);
-        });
+        createRecipe(recipe)
+            .then((result) => {
+                const id = result.data?.library?.createRecipe.id;
+                id
```

Comment:
How could this `id` ever be false-y, other than if the new recipe was assigned ID `0`?

Reply:
I was thinking that sometimes graphQL sends back a 200 and a partial response so it's better to be defensive...but honestly I have no idea if that is even possible in this instance so I can remove if unneeded.

Reply:
Wouldn't apollo have converted the "HTTP 200 w/ some errors in the JSON" into a promise rejection?

Reply:
That is a great question and I am realizing I am not up to speed on the apollo server error handling range. I removed this in favor of simplicity and now have on my to do list to read up on error handling for apollo server and client.

Reply:
I think this is the kind of thing that made me gun-shy of making assumptions: 

<img width="750" alt="image" src="https://github.com/folded-ear/gobrennas-client/assets/363095/514873b8-d717-48c5-aaef-598ff42e0d0d">

Reply:
Thank God we don't use Apollo Server!